### DISCUSSION TITLE
Giscus localization

### DISCUSSION THREAD

**Discussion by ngurmen** [vuepress-theme-hope/vuepress-theme-hope#2392] [Q&A] [answered]
Hi,

Recently, we submitted a PR for Turkish language support.  It was merged and it is working nicely for the most part.  It seems the comments plugin using Giscus is not translated though as shown below.  What files need to be updated to add tr support to Giscus strings?  Thank you.

![image](https://user-images.githubusercontent.com/70996543/200669781-bfe3ac85-ece3-41a7-a982-8bad0a378e94.png)

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/67cd729f3f237d14ec9c0dd589c4260f9ef38c01/packages/comment2/src/client/components/Giscus.ts#L96

Check here.

Edited: Oops, I just missed tr, hhh, you can open a PR to add it.
